### ISSUE TITLE
Cannot train on gpus, only on cpu.

### ISSUE THREAD

**Issue by x12901** [TannerGilbert/Tensorflow-Object-Detection-API-train-custom-Mask-R-CNN-model#4]
I have 4 gpus and can't set batch_size: 1, at least 4. This will cause a memory shortage and I don't have any good solution.

gpu :1080TI *4
tensorflow : 2.3.0
python: 3.6

**Comment by TannerGilbert** [comment]
The higher you set the batch size the better.

**Comment by x12901** [comment]
Shut down the other gpu and it works now.
